calico/netlify.toml

```diff
@@ -4,7 +4,7 @@
   command = '''
 
   cp netlify/Gemfile Gemfile && bundle install
-  make bin/helm bin/helm3
```

Comment:
helm no longer required to build the docs, but we do pre-build a tgz with the ocp manifests in it and copy that in.